SalesforceMobileSDK-iOS.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
       smartstore.dependency 'SalesforceMobileSDK-iOS/SalesforceSDKCore'
       smartstore.dependency 'SQLCipher', '~> 3.1'
       smartstore.dependency 'SQLCipher/fts', '~> 3.1'
-      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h', 'external/fmdb/src/fmdb/*.{h,m}'
+      smartstore.dependency 'FMDB', '~> 2.5'
+      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h'
```

Comment:
It does sound like the pain of maintenance currently outweighs the benefits of keeping it standalone. If we get rid of it, we can remove the only static target dependency we have too. I'm okay with that approach.